docs/CONTRIBUTING.md

```diff
@@ -32,3 +32,75 @@ Coming soon!
 After values are pushed the cluster's drone will do a deployment run. Helmfile diff is used targeting `$CLOUD` & `$CLUSTER` before the final deployment of the artifacts.
 
 If you have any meaningful additions to this, please let us know!
+
+## Schema validation
+
+### Guidelines/best practices
+
+We intend to be very exact in our speech regarding schema validation. Currently we perform meta-schema validation on the `values-schema.yaml` file. By having accurate and consistent descriptions (using the `description` keyword), we can provide the user with a better experience in his configuration management of the `otomi-values` repository.
+
+[Under the Apache V2.0 license agreement we are free to distribute the following content:](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#validation)
```

Comment:
I don't understand why you bring this in here

Reply:
I thought it would be more professional respecting the copyright after copy-and-paste